app/src/androidTest/java/com/android/voyageur/ui/search/SearchScreenTest.kt

```diff
@@ -107,7 +107,7 @@ class SearchScreenTest {
   @Test
   fun testNoResultsFoundPlaces() {
     val searchQuery = "test"
-    `when`(placesRepository.searchPlaces(eq(searchQuery), any(), any())).thenAnswer {
+    `when`(placesRepository.searchPlaces(eq(searchQuery), any(), any(), any())).thenAnswer {
       val onSuccess = it.arguments[1] as (List<Place>) -> Unit
       onSuccess(emptyList())
     }
```

Comment:
Can we do more thorough testing? If possible I suggest testing the different possibilities concerning the location permissions (the popup, what happens when the location is not enabled, etc ...)

Reply:
I thought about this and for the permissions it is really trick to create tests as I am using a system activity that can't be mocked.

I think the current tests are sufficient for the search as the location doesn't matter in the tests as we are mocking that anyways.

Reply:
After we finish this feature completely (#103 is solved) I think we can add tests for the whole repository. Wdyt?

Reply:
Oh ok. Then leave it as is, if our overall line coverage is above 80% then it is fine.